plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-products-controller.php

```diff
@@ -1721,4 +1721,43 @@ public function get_suggested_products( $request ) {
 
 		return parent::get_items( $request );
 	}
+	/**
+	 * Check if a SKU exists.
+	 * @param string $sku SKU.
+	 * @return bool
+	 */
+	public function does_sku_exist( $sku ) {
```

Comment:
We should replace the method `is_existing_sku` in the data store CPT, instead of adding a new method with same functionality in a controller.